godot-core/src/registry/property.rs

```diff
@@ -517,6 +517,7 @@ pub mod export_info_functions {
     // right side are the corresponding property hint. Godot is not always consistent between the two, such
     // as `export_multiline` being `PROPERTY_HINT_MULTILINE_TEXT`.
     default_export_funcs!(
+        export_storage => NONE,
```

Comment:
I was wondering why there's no hint for this, maybe add brief comment:
```suggestion
        export_storage => NONE, // Expressed via property *usage*.
```